frontend/src/components/pages/CreateReview/CreateReviewPage.tsx

```diff
@@ -0,0 +1,276 @@
+            fullName: "Publisher Fisher",
+            publishYear: 2012,
+          },
+        ],
+        seriesName: "The Amazing Series",
+      },
+    ]);
+  }, []);
+
+  return (
+    <Box>
+      {/* <BookModal
+        isOpen={showBookModal}
+        onClose={onBookModalClose}
+        tagsSelected={tagsSelected}
+        handleSelected={handleTagSelected}
+        booksAdded={books}
+        handleBooksAdded={setBooks}
+      /> */}
+      <DeleteModal
+        isOpen={showDeleteBookModal}
+        onClose={onDeleteBookModalClose}
+        bookIndex={deleteBookIndex}
+        deleteBook={deleteBook}
+      />
+      <PublishModal
+        isOpen={showPublishModal}
+        onClose={onPublishModalClose}
+        publishBook={() => {}}
+      />
+      <DeleteReviewModal
+        isOpen={showDeleteReviewModal}
+        onClose={onDeleteReviewModalClose}
+        deleteReview={() => {}}
+      />
+      {/* Tool bar */}
+      <Box
+        display="flex"
+        flexDirection="row"
+        alignItems="center"
+        justifyContent="space-between"
+        m="0"
+        mb="60px"
+        h="60px"
+        w="100%"
+        bgColor="#F6F6F6"
+        boxShadow="lg"
+        padding="0 8% 0 8%"
+      >
+        {/* Contains back button and title */}
+        <Box display="flex" flexDirection="row" alignItems="center">
+          <IconButton
+            aria-label="Go back"
+            variant="ghost"
+            isRound
+            icon={<ArrowBackIcon w={7} h={7} />}
+            mr={6}
+            onClick={() => window.history.back()}
+          />
+          <Text fontSize="18px" fontWeight="semibold">
+            Back to Dashboard
+          </Text>
+        </Box>
+        {/* Contains buttons */}
+        <Box display="flex" flexDirection="row" alignItems="center">
+          <ButtonGroup spacing={6}>
+            <Button variant="ghost">Preview</Button>
+            <Button variant="ghost">Save</Button>
+            <Button
+              colorScheme="teal"
+              bg="#0EBCBD"
+              variant="solid"
+              onClick={() => setShowPublishModal(true)}
+            >
+              Publish
+            </Button>
+            <IconButton
+              aria-label="Discard"
+              variant="ghost"
+              icon={<DeleteIcon w={6} h={6} color="#718096" />}
+              onClick={() => setShowDeleteReviewModal(true)}
+            />
+          </ButtonGroup>
+        </Box>
+      </Box>
+      {/* Main page content */}
+      <Box display="flex" flexDirection="column" m="0px auto" w="70%">
+        <Heading size="lg">Book Information</Heading>
+
+        {/* Add new book and book list */}
+        <Box
+          display="flex"
+          flexDirection="row"
+          alignItems="flex-start"
+          justifyContent="flex-start"
+          mt="10px"
+          mb="50px"
+        >
+          {/* Current books display */}
+          {books.map((book, i) => (
+            <SingleBook
```

Comment:
Theoretically, these should be displayed sorted by `seriesOrder`. That might be better done in another PR though, what do you think?